packages/subgraph/src/utils/erc165.ts

```diff
@@ -3,7 +3,7 @@ import {ethereum, Bytes} from '@graphprotocol/graph-ts';
 export function supportsInterface(
   contract: ethereum.SmartContract,
   interfaceId: string,
-  expected: boolean = true
+  expected = true
```

Comment:
Is this an improvement?

Reply:
yes, if a default value is assigned it get the type from it
Type boolean trivially inferred from a boolean literal, remove type annotation  @typescript-eslint/no-inferrable-types